### PR TITLE
Update Kotlin version and don't run tests when deploying

### PR DESCRIPTION
**Pull request by gnawinganimal**
fix documentation and update kotlin

### CODE REVIEW

File: src/main/java/org/sert2521/gamename/driving/TankDrive.kt [hunk 1]
```diff
@@ -20,3 +20,5 @@ class TankDrive(private val drive: TankDrive,
 
     override fun end() = drive.stop()
 }
+
+
```
Comment (andrewda) [inline src/main/java/org/sert2521/gamename/driving/TankDrive.kt]: Can you delete these extra 2 lines

**Review by andrewda — CHANGES_REQUESTED** [review]

**Comment by andrewda** [comment]
Also sorry to be really picky, but could you separate this into multiple PR's. For something this small it probably doesn't matter _too_ much, but I want us to get in the habit of making PR's for only one fix at a time so we can have a clean version history and divide the work nicer.

File: src/main/java/org/sert2521/gamename/driving/TankDrive.kt (hunk 1 above)
Comment (andrewda) [reply src/main/java/org/sert2521/gamename/driving/TankDrive.kt]: This is why Travis is failing.

**Review by andrewda — APPROVED** [review]
thx @SUPERCILEX

**Comment by SUPERCILEX** [comment]
@andrewda Triggered. I was so close to updating the PR name.

**Comment by SUPERCILEX** [comment]
Dang it! 😆

**Comment by SUPERCILEX** [comment]
3 seconds apart

**Comment by andrewda** [comment]
:joy:

**Pull request merged**